web/src/containers/StateDashboardSidebar.js

```diff
@@ -13,36 +13,39 @@ class Sidebar extends Component {
     const { apds, place } = this.props;
 
     return (
-      <div className="ds-l-col--3">
-        <aside className="site-sidebar bg-teal">
+      <div className="ds-l-col--3 bg-white">
+        <aside className="site-sidebar">
           <div className="xs-hide sm-hide">
-            <div className="px2 py3 lg-px3 lg-py4 bg-white flex items-center">
+            <div className="flex items-center">
               <img
                 src={`/static/img/states/${place.id}.svg`}
                 alt={place.name}
                 className="align-middle mr2"
                 width="40"
                 height="40"
               />
-              <h1 className="m0 blue h3 light caps line-height-2">
+              <h1 className="text-xl">
```

Comment:
Is this a class from the new system or old?

Reply:
This is a class we made up when we were trying to come up with a logical way to break text sizes away from their heading levels (so we could do away with classes named `h2`, for instance).